test/init.js

```diff
@@ -153,6 +158,66 @@ test('init: should add a valid source map for css if wished', function(t) {
   }).write(makeFileCSS());
 });
 
+test('init: can replace `identityMap` option with sourcemap.identityMap stream (js file)', function(t) {
+  var file = makeFile();
+
+  function assert(files) {
+    t.ok(files[0], 'should pass something through');
+    t.ok(files[0] instanceof File, 'should pass a vinyl file through');
+    t.ok(files[0].sourceMap, 'should add a source map object');
+    t.equal(String(files[0].sourceMap.version), '3', 'should have version 3');
+    t.equal(files[0].sourceMap.sources[0], 'helloworld.js', 'should add file to sources');
```

Comment:
@phated this looks good to me. However, is there anyway to make the specific test lines stand out which are specifically relevant to identityMap.

IE: Which of these tests is most indicative that identityMap option worked.

```
 function assert(files) {
 t.ok(files[0], 'should pass something through');
 t.ok(files[0] instanceof File, 'should pass a vinyl file through');
 t.ok(files[0].sourceMap, 'should add a source map object');
 t.equal(String(files[0].sourceMap.version), '3', 'should have version 3');
 t.equal(files[0].sourceMap.sources[0], 'test.css', 'should add file to sources');
 t.equal(files[0].sourceMap.sourcesContent[0], sourceContentCSS, 'should add file content to sourcesContent');
 t.deepEqual(files[0].sourceMap.names, [], 'should add correct names');
 t.equal(files[0].sourceMap.mappings, 'CAAC;GACE;GACA', 'should add correct mappings');
}
```